src/darwin/Framework/CHIP/templates/MTRClusterNames-src.zapt

```diff
@@ -0,0 +1,88 @@
+{{> header excludeZapComment=true}}
+
+
+#import <Foundation/Foundation.h>
+
+#import "MTRClusterNames.h"
+
+@implementation MTRClusterAttributeIDType
+
+#pragma mark - Clusters IDs
+
+- (NSString *)formatMTRClusterIDType:(MTRClusterIDType)clusterID
+{
+    NSString * result = nil;
+    
+    switch (clusterID) {
```

Comment:
I wonder if a lookup-table based solution would be better than a large switch statement.

Reply:
https://github.com/project-chip/connectedhomeip/blob/master/src/lib/format/BUILD.gn#L37 for example contains a lookup for all structures ... although it probably contains too much as it has up to full structure member naming.